c/cert/test/rules/CON30-C/test.c

```diff
@@ -0,0 +1,98 @@
+#include <stdio.h>
```

Comment:
do we expect this to be compliant? currently is reported.
```
void Test() {
  tss_create(&k, NULL); // COMPLIANT
  int *value =
      tss_get(k);
  free(value);
}
```

Reply:
I reworked these test cases -- please have another look @knewbury01